src/pages/fr/core-concepts/framework-components.md

```diff
@@ -0,0 +1,149 @@
+
+## Utilisation des composants de Framework
+
+Utilisez vos composants de Framework JavaScript dans vos composants Astro, Pages et Layouts comme des composants Astro classiques ! Tous vos composants peuvent être placés dans `/src/components`, ou peuvent être organisés de la manière que vous le souhaitez.
+
+Pour utiliser un composant de Framework, importez-le à partir de son chemin relatif (y compris l'extension de fichier) dans le script du composant Astro. Ensuite, utilisez le composant avec d'autres composants, des éléments HTML et des expressions similaire au JSX dans le modèle du composant.
+
+```astro
+---
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<html>
+  <body>
+    <h1>Utilisez des composants React directement dans Astro !</h1>
+    <MyReactComponent />
+  </body>
+</html>
+```
+
+> 💡 _N'oubliez pas : Toutes les importations doivent être **en haut** de votre script de composant Astro._
+
+Par défaut, vos composants de Framework seront rendus en HTML statique. C'est utile pour les composants qui ne sont pas interactifs et qui évite de transmettre à l'utilisateur du JavaScript inutile.
+
+## Hydratation des composants interactifs
+
+Un composant de Framework peut être hydraté en utilisant une directive `client:*`. C'est un attribut de composant pour définir comment votre composant devrait être **rendu** et **hydraté**.
+
+Cette [directive `client:*`](/fr/reference/directives-reference/#client-directives) définit si oui ou non votre composant doit être rendu au moment de la compilation, et quand votre composant doit être chargé par le navigateur, côté client.
+
+La plupart des directives rendront le composant sur le serveur à la compilation. Le JS du composant sera envoyé au client en fonction de la directive spécifiée. Le composant sera hydraté quand son JS aura terminé l'importation.
+
+```astro
+---
+// Exemple: hydratation des composants de Framework dans le navigateur.
+import InteractiveButton from '../components/InteractiveButton.jsx';
+import InteractiveCounter from '../components/InteractiveCounter.jsx';
+---
+<!-- Le JS du composant commencera à importer au chargement de la page -->
+<InteractiveButton client:load />
+
+<!-- Le JS du composant ne sera pas envoyé au client tant que l'utilisateur
+     ne défile pas vers le bas et que le composant est visible sur la page -->
+<InteractiveCounter client:visible />
+```
+
+>⚠️ Tout le JS de rendu nécessaire au Framework (par exemple React, Svelte) est téléchargé avec la page. Les directives `client:*` définissent seulement quand le _JS du composant_ est importé et quand le _composant_ est hydraté.
+
+### Directives d'hydratation disponibles
+
+Il y a plusieurs directives d'hydratation disponibles pour les composants de Framework : `client:load`, `client:idle`, `client:visible`, `client:media={QUERY}` et `client:only=" "`.
```

Comment:
I presume that `client:only=" "` should be `client:only={FRAMEWORK}` to match the English page.